packages/@lwc/wire-service/src/index.ts

```diff
@@ -20,8 +20,9 @@ import {
     Context,
     WireContext,
     WireEventTarget,
-    ValueChangedEvent,
 } from './wiring';
+import { ValueChangedEvent } from './value-changed-event';
+import { LinkContextEvent } from './link-context-event';
```

Comment:
Are these used in this file? If not, can we just do:

```
export { ValueChangedEvent } from './value-changed-event';
export { LinkContextEvent } from './link-context-event';
```

instead of exporting further down the file?

Reply:
they are exported on the last line, and used in the middle to check `instanceof` them.